content/docs/details/scaffolder/index.md

```diff
@@ -0,0 +1,403 @@
+---
+title: Scaffolder
```

Comment:
Should this title be a bit more descriptive? What am I going to learn about the scaffolder?

Reply:
How about now?